compile_testing.py

```diff
@@ -0,0 +1,67 @@
+import os, shutil
+
+excluded_folders = [
+    "multimodal_clip"
+]
+
+try:
+    os.mkdir("testing-folder")
+except:
+    pass
+
+root = "./examples"
+
+requirements = []
+commands = []
+
+# For each folder in the examples folder
+for folders in os.listdir(root):
+    folder = os.path.join(root, folders)
+    if os.path.isdir(folder) and folders not in excluded_folders:
+        for file in os.listdir(folder):
+            # If the file is a requirements.txt file, add it to the list of requirements
+            if file.endswith("requirements.txt"):
+                with open(os.path.join(folder, file), "r") as f:
+                    requirements.extend(f.read().splitlines())
+            # If the file is a python file, copy it to the testing folder
+            if file.endswith(".py"):
+                dup_name = folder.split("/")[-1] + ".py"
+                shutil.copy(os.path.join(folder, file), os.path.join("./testing-folder", dup_name))
+            # If the file is a jupyter notebook, convert it to a python file and copy it to the testing folder
+            # if file.endswith(".ipynb"):
+            #     dup_name = folder.split("/")[-1] + "_IPYNB"
+            #     with open("convert-ipynb.sh", "a+") as f:
+            #         f.write("jupyter nbconvert --output " + dup_name + " --output-dir=\"./testing-folder\" --RegexRemovePreprocessor.patterns=\"^[!%]\" --to python " + os.path.join(folder, file) + "\n")
+            # If the file is a README.md file, add the dataset download commands to the list of commands
+            if file.endswith("README.md"):
```

Comment:
This section might also be not needed if test.py contains everything

Reply:
Same as above, this file is not used

Reply:
So is it okay to get rid of this file if it's not being used?

Reply:
I guess, but it might be useful in the future, which is why I just left it in there.

Reply:
Okay let's remove the files that are not used then. It'll reduce the scope of this PR a bit. If it can be useful in the future, let's keep it in a branch or something but not in main

Reply:
👍 removed